bundler/man/bundle-config.1

```diff
@@ -42,7 +42,7 @@ Executing \fBbundle config set <name> <value>\fR will set that configuration to
 Executing \fBbundle config set \-\-global <name> <value>\fR works the same as above\.
 .
 .P
-Executing \fBbundle config set \-\-local <name> <value>\fR will set that configuration to the local application\. The configuration will be stored in \fBapp/\.bundle/config\fR\. If \fBBUNDLE_APP_CONFIG\fR is set, the configuration will be stored in \fB$BUNDLE_APP_CONFIG/config\fR\.
```

Comment:
I noticed the inconsistent usage of `myproject` vs `yourproject`. I would use something like `<project_root>`, with the `<>` signs to emphasize that it's a placeholder.

Reply:
Could you fix that and rebase this PR while you're at it?

Reply:
Good catch, fixed

Reply:
Thanks!